### PR TITLE
Feat: upload delete popup

### LINKED ISSUE #110: [Task]: Upate delete candy machine to new design
**Task description.**
Move delete to popup like the design states
image
image

**Requirements.**
- Add button
- Move delete to popup
- Functionality works

### PR DESCRIPTION
**Pull request by cpl121**
Fixes #110

## Changes

  - Add popup to delete candy machine
  - Move delete to popup
  - Functionality works
  
### Checks

* [X] Have you catchup your branch with the latest state in base?
* [X] Have you lint your code locally prior to submission?
* [X] Have you reviewed your proposed changes and removed debris?
* [X] Have you successfully ran due tests with your changes?

### CODE REVIEW

File: components/Popup.tsx [hunk 1]
```diff
@@ -24,9 +24,9 @@ const Popup: FC<{
                 sx={{
                     zIndex: 10,
                     width: '100%',
-                    maxWidth: '60vh',
+                    maxWidth: size === 'large' ? '60vh' : '45vh',
                     height: '100%',
-                    maxHeight: size === 'large' ? ['550px', '750px'] : ['50%'],
+                    maxHeight: size === 'large' ? ['550px', '750px'] : ['27.5%'],
```
Comment (evavirseda) [inline components/Popup.tsx:29]:
```suggestion
                    maxHeight: size === 'large' ? ['550px', '750px'] : ['40%', '27.5%'],
```

File: components/DeleteCandyMachine.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            setStatus({ error: true, message: 'Delete was not successful.' })
+        }
+        setIsDeleting(false)
+    }
+
+    return (
+        <>
+            <div style={{ display: 'grid', gridRowGap: '5px' }}>
+                <Text as='p'>Are you sure you want to delete candy machine {candyMachineAccount}</Text>
+                {!isDeleting && status.message && (
```
Comment (evavirseda) [inline components/DeleteCandyMachine.tsx:36]: :woozy_face: 
image

**Review by evavirseda — CHANGES_REQUESTED** [review]

**Comment by VmMad** [comment]
image
This is happening to me.
On low height screens this happens too... 
image
Change button title "Delete candy machine" to "Delete" 
Also, the disabled button when you click delete is bigger than the previous one, it looks a little weird

**Comment by cpl121** [comment]
> image This is happening to me. On low height screens this happens too... image Change button title "Delete candy machine" to "Delete" Also, the disabled button when you click delete is bigger than the previous one, it looks a little weird

Yes, this is solved in [this pr](https://github.com/boxfish-studio/sugar-rush/pull/148) because the behavior is similar

**Comment by agus-xyz** [comment]
please @cpl121 lets fix the conflicts with the base branch before proceeding with a final review 🙏🏼

**Pull request merged**